hphp/hack/hhi/stdlib/builtins_class.hhi

```diff
@@ -12,17 +12,17 @@ function get_declared_classes();
 function get_declared_interfaces();
 function get_declared_traits();
 function enum_exists(string $class_name, bool $autoload = true): bool;
-function class_exists($class_name, bool $autoload = true): bool;
-function interface_exists($interface_name, bool $autoload = true): bool;
-function trait_exists($trait_name, bool $autoload = true): bool;
-function get_class_methods($class_or_object);
-function get_class_vars($class_name);
+function class_exists(string $class_name, bool $autoload = true): bool;
```

Comment:
If you've got a `classname`, then the class exists so there's no point calling this.

I'm using `class_exists` as I can't type user attributes, so can't enforce something being a `classname`, so have to manually check the value is valid rather than relying on the type checker.